tests/cargo-kani/stubbing-ws-packages/top/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+[package]
+name = "top"
```

Comment:
The name is very hardware-ish :)

Reply:
LoL... I did not even think about it. Any other suggestion?

Reply:
No, just pointing it out :)